GitUI/CommandsDialogs/FormBrowse.Designer.cs

```diff
@@ -221,26 +222,32 @@ private void InitializeComponent()
             // toolStripMenuItem2
             // 
             toolStripMenuItem2.Name = "toolStripMenuItem2";
-            toolStripMenuItem2.Size = new System.Drawing.Size(122, 22);
+            toolStripMenuItem2.Size = new System.Drawing.Size(184, 34);
```

Comment:
The commit that moves to the status bar also increases the size for icons in the button bar, looks ugly. Designer...

Reply:
I'll double check but I am unable to see any artifacts locally.

Reply:
Should be
![image](https://user-images.githubusercontent.com/6248932/57588506-6cb3da80-7515-11e9-9ce5-09cbae24b4f4.png)

Is at 100%
![image](https://user-images.githubusercontent.com/6248932/57588510-7e957d80-7515-11e9-93b3-eb48d3a35d94.png)